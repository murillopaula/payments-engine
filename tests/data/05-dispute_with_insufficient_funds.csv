type,client,tx,amount
deposit,65535,4294967295,1.0
withdrawal,65535,1,0.5
dispute,65535,4294967295,
chargeback,65535,4294967295,
