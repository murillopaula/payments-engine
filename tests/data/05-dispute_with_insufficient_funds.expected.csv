client,available,held,total,locked
65535,0.5000,0.0000,0.5000,false
